ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4533,4 +4533,14 @@ AHU_NON_CANONICAL_TYPE_3:
   - EDPM
   - EFSS
   - RTM
-  - ZHM
\ No newline at end of file
+  - ZHM
+
+AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_ZHHC:
```

Comment:
Please add `_NONCANONICAL_1` to end of name. This type name should be reserved for the canonical combination of these implemented abstract types.